src/platform/site-wide/mega-menu/components/SubMenu.jsx

```diff
@@ -40,49 +39,47 @@ const SubMenu = ({
         id={id}
         role="group"
       >
-        <FocusLock disabled={!mobileMediaQuery?.matches}>
-          <div>
-            <button
-              className="back-button"
-              aria-controls={`vetnav-${_.kebabCase(navTitle)}`}
-              onClick={() => handleBackToMenu()}
+        <div>
+          <button
+            className="back-button"
+            aria-controls={`vetnav-${_.kebabCase(navTitle)}`}
+            onClick={() => handleBackToMenu()}
+          >
+            Back to Menu
+          </button>
+        </div>
+
+        {seeAllLink && (
+          <div className="panel-bottom-link">
```

Comment:
Same here 🙂

Reply:
This div is actually important for styling purposes. The same class on the `a` doesn't have the same effect.

Reply:
Perfect, thanks for trying! 🙂